configs/textrecog/seg/convert.py

```diff
@@ -0,0 +1,112 @@
+import cv2
+import torch
+import torch.nn as nn
+from collections import OrderedDict
+import os
+import numpy as np
+from argparse import ArgumentParser
+
+import mmcv
+from mmcv.parallel import collate, scatter
+from mmcv.runner import load_checkpoint
+from mmdet.apis import init_detector
+from mmdet.datasets import replace_ImageToTensor
+from mmdet.datasets.pipelines import Compose
+
+from mmocr.apis.inference import model_inference
+from mmocr.datasets import build_dataset  # noqa: F401
+from mmocr.models import build_detector  # noqa: F401
+import mmocr.models.textrecog.backbones as K
+
+config = './seg_r31_1by16_fpnocr_academic.py'
+checkpoint = './seg_r31_1by16_fpnocr_academic-72235b11.pth'
+if isinstance(config, str):
+    config = mmcv.Config.fromfile(config)
+
+elif not isinstance(config, mmcv.Config):
+    raise TypeError('config must be a filename or Config object, '
+                    f'but got {type(config)}')
+
+config.model.pretrained = None
+config.model.train_cfg = None
+model = build_detector(config.model, test_cfg=config.get('test_cfg'))
+state_dict = torch.load(
+    checkpoint, map_location=lambda storage, loc: storage)
+# model.CLASSES = state_dict['meta']['CLASSES']
+new_state_dict = OrderedDict()
+
+for k, v in state_dict.items():
+    new_state_dict.update(v)
+
+del [new_state_dict['CLASSES']]
+# # # load params
+# print(new_state_dict.keys())
+model.load_state_dict(new_state_dict)
+model.eval()
+# # convert pth-model to pt-model
+# example = torch.rand(1, 1, 32, 512).cpu()
+# traced_script_module = torch.jit.trace(model, example)
+# traced_script_module.save("./src/crnn.pt")
+
+model.to(device='cuda:0')
```

Comment:
Use argparse if cpu is also supported

Reply:
Sorry，This is just the test version I modified, so many places are written for convenience. When I am free, I will write more formally